lib/ruby_lsp/tapioca/addon.rb

```diff
@@ -63,7 +69,41 @@ def workspace_did_change_watched_files(changes)
         return unless T.must(@global_state).enabled_feature?(:tapiocaAddon)
         return unless @rails_runner_client # Client is not ready
 
-        nil
+        constants = changes.flat_map do |change|
+          path = URI(change[:uri]).to_standardized_path
```

Comment:
Perhaps extract the body of this to its own method for readability?

Reply:
I'm a little uncomfortable that `file_updated?` has side-effects. I'll take a look to see if there is another way we could structure it.